### PR TITLE
Payment Integration with Razorpay

### PR DESCRIPTION
**Pull request by priyanshm07**
# Pull Request

## What does this PR do?
Feature #377

## What part does this affect?
- [x] FrontEnd.
- [ ] BackEnd.
- [ ] Documentation.
- [ ] Other. (Please specify below)

## Before submitting
- [ ] Was this discussed/approved via a GitHub issue or slack?
- [x] Did you read the [contributor guideline](https://github.com/Auto-DL/Auto-DL/blob/v1-beta/CONTRIBUTING.md)?
- [x] Did you ensure that there aren't any other open [Pull Requests](https://www.github.com/Auto-DL/Generator/pulls) for the same update/change?
- [x] Did you make sure the title is self-explanatory and the description concisely explains the PR?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure the code is clean and docstrings have been added or updated as required?
- [x] Did you make sure the code is linted/formatted locally prior to submission? (using `black` and/or `prettier`)
- [x] Did you make sure to update the documentation with your changes? (if necessary)

## PR review
Anyone in the community is free to review the PR once the tests have passed.

Thank you for contributing to [AutoDL](https://github.com/Auto-DL/Auto-DL). We look forward to your continued support.

### CODE REVIEW

File: BackEndApp/authv1/views.py [hunk 1]
```diff
@@ -40,15 +59,19 @@ def login(request):
...
         username = request.data.get("username")
+        # username = "priyansh"
         email = request.data.get("email")
+        # email = "priyansh@gmail.com"
         password = request.data.get("password")
+        # password = "priyansh"
         first_name = request.data.get("first_name")
+        # first_name = "priyansh"
         last_name = request.data.get("last_name")
+        # last_name = "maheshwari"
```
Comment (RusherRG) [inline BackEndApp/authv1/views.py:74]: @priyanshm07 please remove the commented lines. There are few more files that have commented lines. Please remove all of them

File: FrontEndApp/v1-react/src/Components/MainApp/Layout.js [hunk 2]
```diff
@@ -120,6 +125,72 @@ function Layout() {
...
+        "Content-Type": "application/json",
+      },
+    }).then((res) => {
+      console.log("Data: ",res)
+      return res;
+    });
+
+
+    var options = {
+        key: 'RAZORPAY_API_KEY', // Enter the Key ID generated from the Dashboard
```
Comment (RusherRG) [inline FrontEndApp/v1-react/src/Components/MainApp/Layout.js:165]: Load the key using `process.env`

File: FrontEndApp/v1-react/src/Components/Login/LoginForm.js [hunk 3]
```diff
@@ -195,7 +196,9 @@ function LoginForm() {
       values.email &&
       values.registerPassword === values.confirmpassword
     ) {
+      console.log("Validate: ", data)
       const res = await LoginService.register(data);
+      console.log("Register Response: ",res)
```
Comment (RusherRG) [inline FrontEndApp/v1-react/src/Components/Login/LoginForm.js:201]: Remove the `console.logs`

**Review by RusherRG — CHANGES_REQUESTED** [review]

File: BackEndApp/authv1/views.py (hunk 1 above)
Comment (priyanshm07) [reply BackEndApp/authv1/views.py:74]: @RusherRG ok, am updating it.

**Comment by RusherRG** [comment]
@priyanshm07 Thanks for creating the pull request. We'll test out the code and let you know if there is need to make any changes. In the meanwhile, make the requested code changes

**Comment by priyanshm07** [comment]
> @priyanshm07 Thanks for creating the pull request. We'll test out the code and let you know if there is need to make any changes. In the meanwhile, make the requested code changes

@RusherRG Well thanks for the support

**Pull request closed without merging**